docusaurus/docs/create-a-plugin/develop-a-plugin/manage-go.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: backend
-title: Backend
+id: manage-go
+title: Manage Go for backend
 sidebar_position: 2
 ---
 
```

Comment:
Suggestion for below -
The following commands are frequently used during backend plugin development:

Reply:
Fixed.